tools/pyi/gen_pyi.py

```diff
@@ -1426,33 +1659,31 @@ def replace_special_case(hint: str) -> str:
         "torch/_C/__init__.pyi",
         "torch/_C/__init__.pyi.in",
         lambda: {
-            "generated_comment": "@" + "generated from torch/_C/__init__.pyi.in",
+            "generated_comment": "@generated from torch/_C/__init__.pyi.in",
```

Comment:
You can't do this, this will make editors think that gen_pyi.py is a generated file and refuse to edit it

Reply:
I do not quite understand this because the string is identical. I revert this change anyway.

Reply:
The point is that editors just blindly grep for @generated in a file to figure out if it's generated. But the codegen script is not generated, so you need to avoid the grep matching it